lua/lspconfig/configs.lua

```diff
@@ -1,6 +1,6 @@
 local util = require 'lspconfig.util'
 local async = require 'lspconfig.async'
-local api, validate, lsp, uv, fn = vim.api, vim.validate, vim.lsp, (vim.uv or vim.loop), vim.fn
+local api, validate, lsp, uv, fn = vim.api, vim.validate, vim.lsp, vim.uv, vim.fn
```

Comment:
does 0.9 have vim.uv ?

Reply:
No. I was just testing if the CI catches this fact. Should've done this in private but was too lazy. Will close PR after confirmation.